pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -303,6 +309,83 @@ func CreateMemberToken(cmd *cobra.Command, k8sClient client.Client, name string,
 	return nil
 }
 
+func DeleteMemberToken(cmd *cobra.Command, k8sClient client.Client, name string, namespace string) error {
+	errFunc := func(kind string, act string, err error) error {
+		if apierrors.IsNotFound(err) {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s %s not found in Namespace %s\n", kind, name, namespace)
+			return nil
+		}
+		if err != nil {
+			return err
+		}
+		if act == "delete" {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s %s is deleted\n", kind, name)
```

Comment:
```suggestion
			fmt.Fprintf(cmd.OutOrStdout(), "%s %s deleted\n", kind, name)
```

Reply:
@luolanzone seems like if we have the "is" then it becomes a completed sentence, or this is too short?

Reply:
I was taking kubectl output as a reference. You may check other commands' output to keep them consistent.